boilerplates/react-union-boilerplate-liferay-basic/hero-portlet/src/main/java/eu/reactunion/boilerplate/portlet/data/InitData.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) Lundegaard, s.r.o. 2018 - All Rights Reserved
+ * Proprietary and confidential. Unauthorized copying of this file, via any medium is strictly prohibited.
+ */
+package eu.reactunion.boilerplate.portlet.data;
+
+/**
```

Comment:
Class comment if if its obvious.

Reply:
hero-portlet folder was removed, but javadoc was added to InitData class in content portlet.